ledger/participant-integration-api/src/main/scala/platform/apiserver/services/LedgerConfigProvider.scala

```diff
@@ -164,11 +164,9 @@ private[apiserver] final class LedgerConfigProvider private (
         .map {
           case state.SubmissionResult.Acknowledged =>
             logger.info(s"Initial configuration submission $submissionId was successful")
-          case state.SubmissionResult.NotSupported =>
```

Comment:
This will fail on canton:
https://github.com/DACH-NY/canton/blob/2334662676d013852bd07d9a90ab44ea56f0aed2/community/participant/src/main/scala/com/digitalasset/canton/participant/sync/CantonSyncService.scala#L323